### PR TITLE
feat: 1478 Keycloak Login Page Capitalization

### PR DESCRIPTION
**Pull request by thegentlemanphysicist**

### CODE REVIEW

File: testing/cypress/e2e/login-page-name-capitalization.cy.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    cy.wait(2000); // Wait a bit because otherwise it will not pick up the value
+
+    cy.get('button').contains('Login').click();
+    cy.wait(2000); // Wait a bit because to make sure the page is loaded
+
+    // On the IDP Select Page, select/test the IDP
+    cy.get('#kc-header-wrapper').contains(data[0].create.ssoheaderdev);
+    cy.get('#kc-social-providers').within(() => {
+      let n = 0;
+      while (n < data[0].create.identityprovider.length) {
```
Comment (arcshiftsolutions) [inline testing/cypress/e2e/login-page-name-capitalization.cy.ts]: Please use the streams API instead of this while loop

**Review by arcshiftsolutions — APPROVED** [review]

**Comment by rstens** [comment]
It looks like your base was an older version of the code. I'd like to do a walkthrough of the code with you to point out a list of minor things.

**Comment by arcshiftsolutions** [comment]
@rstens Can you please indicate your recommended changes in a comment for posterity? That way we can all understand.

**Comment by rstens** [comment]
Will do.:
* createdRequest.json in now called createdrequest.json
* Base verson of the code needs to be updated. Looks like several weeks old.
* Have to find out in whci categor this test sits, all our tests are grouped
* We have some technical Debt with regards to the Playground class, right now we have 4 scripts doing the same thing in 4 different ways
* do not need the cy.wrap on line 16, just req.createRequest() to replace 16-18
* We now have smoke test indicator in our test data and a bit of code in our scripts to govern that

**Pull request merged**